systems/registry/org/pkg/utils.go

```diff
@@ -0,0 +1,114 @@
+package pkg
+
+import (
+	"bytes"
+	"html/template"
+)
+
+func GenerateEmailBody(invitationID string, link string, owner string, org string, role string, name string) (string, error) {
+	bodyTemplate := `
```

Comment:
This string can go inside an *.html.tmpl file.

Reply:
Need your guidance on this.